app/screens/SignupLogin.js

```diff
@@ -82,6 +108,32 @@ export class SignupLogin extends Component {
             Linking.getInitialURL().then((initialUrl) => {
                 if (initialUrl.indexOf('location_id=') > 0) {
                     this.props.navigation.navigate('LocationDetails', { id: initialUrl.split('=')[1]})
+                } else if (url.indexOf('profile') > 0) {
```

Comment:
This should be initialUrl or we need to update after 'then' to be url. I will push up a fix.

Reply:
Ah shoot. Right at the last second I remembered you said it was in two places. So I just copy/pasted without realizing it was a little different.